requirements.txt

```diff
@@ -11,4 +11,6 @@ pytest>=5.4.3,<5.5.0
 coverage>=5.1,<6.0
 Flask-Babel>=1.0.0,<1.1.0
 flake8>=3.8.3,<3.9.0
-python-dotenv>=0.14.0,<0.15.0
\ No newline at end of file
+python-dotenv>=0.14.0,<0.15.0
+psycopg2>=2.8.5,<2.8.6
```

Comment:
for pycopg2 to be installed you have to install these package
```bash
apt-get install postgresql-client gcc libc-dev linux-headers postgresql-dev 
```
or in alpine Dockerfile:

```
RUN apk add --update --no-cache postgresql-client
RUN apk add --update --no-cache --virtual .tmp-build-deps \
    gcc libc-dev linux-headers postgresql-dev 

```